site/assets/scss/_footer.scss

```diff
@@ -3,17 +3,13 @@
 //
 
 .bd-footer {
-  @include font-size(.875rem);
-  position: sticky;
-  top: 100vh;
-  font-weight: 700;
+  // Boosted mod
+  .bd-footer-info {
```

Comment:
Not possible to use `.accordion-header` and get rid of this new class and so get rid of the `.bd-footer` here (and in the HTML) as well?

Reply:
Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1280/commits/bc8db3e877b684e034727d1b5ff0a3eaa8ba8512